### DISCUSSION TITLE
Server side Rendering is not working on production

### DISCUSSION THREAD

**Discussion by sssghulamrasool** [vercel/next.js#59620] [App Router]
Hello,
When I deploy my Next.js application on Vercel, the application is not server-side rendered. What is the problem?
There doesn't seem to be an issue with the local server. I'm not certain what's causing the problem. Can anyone offer assistance?

![image](https://github.com/vercel/next.js/assets/115653920/b30f7323-5413-4c5a-8f73-15bb6fa2740f)

**Comment by icyJoseph** [comment]
Hi,

You are seeing the `__next_error__` UI, which means that the SSR crashed out.

Are you accessing window unguarded? or perhaps using browser APIs that are not available when deploying? Maybe some secrets are missing?
